### ISSUE TITLE
SQL error when importing to an Entries field mapped via custom field

### ISSUE THREAD

**Issue by carlcs** [verbb/feed-me#381]
### Description

I am getting this SQL error when importing to an Entries field that is setup to map via a custom field, in this case a plain text field with handle `agentId`.

This did work fine in beta 18.

```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'agentId' in 'where clause'
The SQL being executed was: SELECT `elements`.`id` FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `elementsSitesId`, `content`.`id` AS `contentId` FROM `elements` `elements` INNER JOIN `entries` `entries` ON `entries`.`id` = `elements`.`id` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id` INNER JOIN `content` `content` ON `content`.`elementId` = `elements`.`id` LEFT JOIN `structureelements` `structureelements` ON `structureelements`.`elementId` = `elements`.`id` WHERE (`entries`.`sectionId`='2') AND (`elements_sites`.`siteId`='1') AND (`agentId` = '4') AND (`content`.`siteId`='1') AND (`elements`.`archived`=FALSE) AND (`elements_sites`.`enabled`=TRUE) ORDER BY `structureelements`.`lft`, `entries`.`postDate` DESC LIMIT 1) `subquery` INNER JOIN `entries` `entries` ON `entries`.`id` = `subquery`.`elementsId` INNER JOIN `elements` `elements` ON `elements`.`id` = `subquery`.`elementsId` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`elementsSitesId` INNER JOIN `content` `content` ON `content`.`id` = `subquery`.`contentId` LEFT JOIN `structureelements` `structureelements` ON `structureelements`.`elementId` = `subquery`.`elementsId` ORDER BY `structureelements`.`lft`, `entries`.`postDate` DESC - Schema.php:664
```

### Additional info

- Plugin version: 3.0.0-beta-21
- Craft version: 3.0.21

**Comment by carlcs** [comment]
@rod-meaney you can downgrade to beta 18 without problems, there were no schema changes since

**Comment by exophunk** [comment]
I have the same issue. Downgrading to beta 18 solves it for now.

**Comment by urbantrout** [comment]
I partially fixed this issue in this commig urbantrout/feed-me@9ee5eae47f2f31037239e5b55b54f6faff2951aa


This is not a complete fix, since the prefix will be prepended for every
field. This should not be the case for e.g. title and all standard
fields.

**Comment by engram-design** [comment]
Sorry all for this issue and delay, fixed in [3.0.0-beta.22](https://github.com/verbb/feed-me/releases/tag/3.0.0-beta.22).

**Comment by carlcs** [comment]
Thanks for the fix. And no worries about the wait, beta 18 was doing it just fine for us.
